simulation.py

```diff
@@ -0,0 +1,90 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import argparse
+import copy
+import importlib
+from thousandaire.constants import DATA_LIST_ALL
+from thousandaire.constants import TRADING_CONFIGS, TRADING_REGIONS
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+COST_OBJECT = 'cost_object'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays_all = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGIONS if region in raw_data['workdays']}
+    bound_data = {}
+    for region, workdays in workdays_all.items():
+        bound_data[region] = copy.deepcopy(raw_data)
+        for dataset in bound_data[region].values():
+            dataset.set_workdays(workdays)
+    bound_data['workdays'] = workdays_all
+    return bound_data
+
+def extract_data(raw_data, data_list, price_dataset, region):
+    """
+    Extract data on data_list from raw_data.
+
+    Return data will be a dictionary, there will be three keys:
+    (1) 'workdays'
+        Mean the trading region. The type of value would be a DataController.
+    (2) 'target_data'
+        Mean the trading target. The type of value would be a Dataset.
+    (3) 'others'
+        Mean the other data in data_list. The type of value would be a
+        dictionary combined with Dataset.
+    """
+    return {
+        'workdays' : raw_data['workdays'][region],
+        'target_data' : raw_data[region][price_dataset],
+        'others' : {
+            name : raw_data[region][name]
+            for name in data_list if name in raw_data[region]}}
+
+def simulate(alpha_settings_path, un_look):
+    """
+    Act the process of simulating.
+    """
+    try:
+        settings = importlib.import_module(alpha_settings_path).AlphaSettings()
+    except FileNotFoundError as error_1:
+        raise FileNotFoundError("Alpha does not exist.") from error_1
+    except ImportError as error_2:
+        raise ImportError("No available AlphaSettings in %s"
+                          % alpha_settings_path) from error_2
```

Comment:
Since these two errors will not happen at the same time, it's actually OK to name both error variables `error`. No need to distinguish them by `error_1` and `error_2` -- putting a number in the names doesn't make them more readable either, indeed.